api/web-accessible-resources/readme.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
I'm assuming this attribution is intended.

Reply:
Yep!